### DISCUSSION TITLE
DrawIO like application with MaxGraph

### DISCUSSION THREAD

**Discussion by asgharhussaindev** [maxGraph/maxGraph#253] [Q&A]
Is there a DrawIO like application made using MaxGraph?

**Comment by tbouffard** [comment]
Hello, what you are looking for was provided in `mxGraph` under the name `GraphEditor`.
This is something that has not yet been migrated to `maxGraph`.

For more details, you can read #99 or #172. There are other places in this repository where the subject is discussed
